src/main/resources/templates/help.html

```diff
@@ -26,8 +25,15 @@
             $("#nav-placeholder").load("/navbar");
         });
     </script>
+    <script>
+        $(document).ready(function() {
+            $("#footer-placeholder").load("/footer");
+        });
+    </script>
+    <title>Help</title>
 </head>
 <body>
-    <div id="nav-placeholder"></div>
+<div id="nav-placeholder"></div>
```

Comment:
does this page (and I think there was another one - chat?) not have any content?

Reply:
we deleted the chat function, but we do not have any content in "Help" for now. Content will be aded when we decided what we want to write

Reply:
ah ok perfect. then just delete the chat one